src/server/api/organization.js

```diff
@@ -206,14 +206,18 @@ export const resolvers = {
     textingHoursStart: organization => organization.texting_hours_start,
     textingHoursEnd: organization => organization.texting_hours_end,
     texterUIConfig: async (organization, _, { user }) => {
-      await accessRequired(user, organization.id, "OWNER");
-      const options = getConfig("TEXTER_UI_SETTINGS", organization) || null;
-      // note this is global, since we need the set that's globally enabled/allowed to choose from
-      const sideboxChoices = getSideboxChoices();
-      return {
-        options,
-        sideboxChoices
-      };
+      try {
+        await accessRequired(user, organization.id, "OWNER");
+        const options = getConfig("TEXTER_UI_SETTINGS", organization) || null;
+        // note this is global, since we need the set that's globally enabled/allowed to choose from
+        const sideboxChoices = getSideboxChoices();
+        return {
+          options,
+          sideboxChoices
+        };
+      } catch (caught) {
```

Comment:
can you move this catch up to just cover the accessRequired?

Reply:
@schuyler1d I made this change.